packages/recomponents/.storybook/theme.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Custom theme
+ */
+import { create } from '@storybook/theming/create';
+
+export default create({
+
+  // UI
+  appBg: 'white',
+
+
+    //Branding
+    brandTitle: 'Rebilly Recomponents',
+    brandUrl: 'https://rebilly.com',
+    brandImage: 'https://media.giphy.com/media/Up1Otz0EISNQcDGmlc/giphy.gif',
```

Comment:
Looks good.
Maybe the animation is spinning and attracting too much attention?

Reply:
Let's give this gif a try. With first complain I can remove it.